desktop/apps/artwork/components/artists/market_insights.js

```diff
@@ -0,0 +1,22 @@
+import $ from 'jquery'
```

Comment:
This file should really have the `jsx` extension, unless that’s not being done through Force?